aws/environments/fxadevtest.yml

```diff
@@ -0,0 +1,18 @@
+---
+region: us-west-2
+subdomain: fxadevtest.dev
+hosted_zone: lcip.org
+ssl_certificate_name: exp20170412_wildcard_dev_lcip.org
+rds_password: ab4b6475c68a8734
+
+ec2_instance_type: t2.medium
+ec2_volume_size: 24
+
+owner: jrgm@mozilla.com
+reaper_spare_me: "true"
+
+fxadev_git_version: docker-changes
+
+# bleh
+cdn_ssl_certificate_id: ASCAIAYDJPL5OT7LFEZSE
+content_static_resource_url: https://static-latest.dev.lcip.org
```

Comment:
Usually values have default, but these ones don't so they have to set to something. Hence, bleh.

TODO: add defaults and drop these lines where we're not going to use Cloudfront.